src/main/resources/db/migration/courtcase/V111__add_offender_matches.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+create table offender_match (id int8 generated by default as identity, case_no varchar(255) not null, confirmed boolean not null, court_code varchar(255) not null, crn varchar(255) not null, cro varchar(255), match_type varchar(255), pnc varchar(255), primary key (id));
```

Comment:
Didn't originally want to commit to mandating that a match had to belong to a group, my rationale being that it would keep our options open. Have locked down the other things though so will change this too so our schema is at least simple and clear about its expectations.